mender-deb-package

```diff
@@ -371,6 +377,8 @@ prepare_recipe
 
 postprocess_recipe
 
+remove_shared_libssp
```

Comment:
I think we should do this from the images instead. Basically [here](https://github.com/mendersoftware/mender-test-containers/blob/0037762f1c2e7bfff6ea90d0afe879b2d1732eb7/mender-dist-packages-building/install-armv6-cross-toolchain#L270). There it is more contextualized, I think.

Otherwise is yet another thing that here needs special handling only for `debian` and only for `armhf` - which is the reason why CI is failing right now.

Reply:
Hmm, I was actually thinking, since the lib is present in the other cross-compilers also, to remove it everywhere.

Reply:
> We should do this from the images instead. Basically [here](https://github.com/mendersoftware/mender-test-containers/blob/0037762f1c2e7bfff6ea90d0afe879b2d1732eb7/mender-dist-packages-building/install-armv6-cross-toolchain#L270). There it is more contextualized, I think.

Yes, good idea!

Reply:
Follow up: https://github.com/mendersoftware/mender-test-containers/pull/481

Reply:
> Hmm, I was actually thinking, since the lib is present in the other cross-compilers also, to remove it everywhere.

Yes, exactly.